tests_metricflow/snapshots/test_derived_metric_rendering.py/SqlQueryPlan/DuckDB/test_nested_filters__plan0_optimized.sql

```diff
@@ -48,71 +46,55 @@ FROM (
         ON
           subq_36.listing = listings_latest_src_28000.listing_id
       ) subq_41
-      WHERE (listing__is_lux_latest) AND (booking__is_instant)
+      WHERE listing__is_lux_latest
```

Comment:
Note - this is an example of removing part of a filter expression - the booking__is_instant filter was applied via pushdown (in the collapsed code section above) on the left side of the join, and we no longer repeat it here.

Reply:
Sweet 😎